spring-cloud-context/src/main/java/org/springframework/cloud/autoconfigure/RefreshEndpointAutoConfiguration.java

```diff
@@ -112,9 +117,15 @@ public RestartEndpoint restartEndpoint() {
 
 		@Bean
 		@ConditionalOnMissingBean
-		public RefreshEndpoint refreshEndpoint(ConfigurableApplicationContext context,
+		public RefreshSupport refreshSupport(ConfigurableApplicationContext context,
```

Comment:
This bean is still guarded by `@ConditionalOnClass(Endpoint.class)`. It should be reworked so it is not.